open-metadata-test/open-metadata-fvt/access-services-fvt/subject-area-fvt/src/test/java/org/odpi/openmetadata/accessservices/subjectarea/fvt/junit/AllIT.java

```diff
@@ -0,0 +1,61 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+
+package org.odpi.openmetadata.accessservices.subjectarea.fvt.junit;
+
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
+import org.odpi.openmetadata.accessservices.subjectarea.fvt.*;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+@Execution(ExecutionMode.SAME_THREAD)
```

Comment:
Thanks - I missed this when looking. Though I know @davidradl is looking at fixing the underlying issue, so will let him decide whether you want to go with this fix or not 

It's good to know how to get them working, so thanks for sharing the annotation.

I think the other aspect is which way @davidradl thinks makes sense to structure the subject area FVTs in future. I was envisioning that perhaps you would gradually move some of the tests into the JUnit classes themselves, which is why I created one test class in parallel with the FVT classes. Of course you can't order that way so understand why you suggest the change (I tried the same but missed that extra annotation!)

It probably depends whether the other fixes are near term or not, but I think we'd all agree we don't want to depend on single thread execution and ordering to allow them to work in future

Will leave to you both to agree. I'm ok with either . See #3312 #3304 #3302  which all seem to have some overlap